src/login.c

```diff
@@ -503,13 +491,21 @@ int main (int argc, char **argv)
 	const char *failent_user;
 	char *host = NULL;
 
-#ifdef USE_PAM
+#if defined(USE_PAM)
 	int retcode;
 	pid_t child;
 	char *pam_user = NULL;
 #else
+# if defined(ENABLE_LASTLOG)
+	char ptime[80];
+# endif
```

Comment:
Move this after the declaration of `struct spwd *spwd = NULL;`. I think it makes it easier to read.